src/tests/BuildWasmApps/testassets/SatelliteAssemblyFromProjectRef/LibraryWithResources/LibraryWithResources.csproj

```diff
@@ -1,5 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Can't this project and the other ones touched under src/tests/ not use the globally defined tfm?

Reply:
These get built on helix and don't have the other props files around.

Reply:
Can't a props file that just contains the tfm be carried along to helix to avoid hardcoding the tfm?

Reply:
We will need to generate one. Yeah, we can do that. I will add it in a follow up PR.

Reply:
That would be best.